src/views/product-root-docs-path-landing/components/paragraph-block/index.tsx

```diff
@@ -0,0 +1,6 @@
+import { MDXRemote } from 'next-mdx-remote'
+import { MdxA, MdxP } from 'components/dev-dot-content/mdx-components'
+
+export function ParagraphBlock(props) {
+	return <MDXRemote {...props.mdxSource} components={{ a: MdxA, p: MdxP }} />
```

Comment:
Hooks into existing `mdx-components` to get things rendering the way we expect. Doesn't do much else!